libraries/lib-stretching-sequence/ClipSegment.cpp

```diff
@@ -94,17 +94,16 @@ void ClipSegment::Pull(float* const* buffers, size_t samplesPerChannel)
       constexpr auto mayThrow = false;
       const auto start =
          forward ? mLastReadSample : mLastReadSample - numSamplesToRead;
-      const auto offset = forward ? 0u : samplesPerChannel - numSamplesToRead;
```

Comment:
I tried to demonstrate backwards scrubbing going wrong without this fix, but I could not.

Maybe `samplesPerChannel` is never greater than `remainingSamplesInClip` because of other tests elsewhere, but the class `ClipSegment` should not depend on that pattern of usage.